arch/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 
 [features]
 default = []
-sev_snp = []
+sev_snp = ["igvm_defs", "igvm_parser"]
```

Comment:
It does not look right to have a feature dependency like this. Should it be `igvm = ["igvm_defs", "igvm_parser"]`?

The current feature dependency suggests that  `"igvm_defs", "igvm_parser"` will only be needed for `sev_snp`. That is they are not needed when the `igvm` feature is enabled. Is this intentional?

Reply:
Yes. Some fileds in the Entrypoint are sev_snp related and the definition is coming from igvm crates. I think some followup commits in another PR will have igvm crate in the arch module.Also in the hypervisor crate, sev_snp guarded code needs definitions from igvm crates.